shared/locales/activities/activities.en-US.properties

```diff
@@ -1,31 +1,50 @@
 # Default activities label
 
-# PICK
-default-activity-pickimage    = Take picture
+# LOCALIZATION NOTE This strings are used to describe actions available for
```

Comment:
"These strings are used to describe available actions that can be associated by default to an application" ?

This -> singular, these -> plural (also in the rest of the comment)